tests/web/test_statuses.py

```diff
@@ -21,6 +23,106 @@ def test_get_transaction_status_endpoint(b, client, user_pk):
 
 
 @pytest.mark.bdb
+def test_get_backlog_transaction_status(b, client, backlog_tx):
+    from bigchaindb import Bigchain
+    from bigchaindb.web.views.statuses import StatusApi
+    from bigchaindb.web.views.transactions import TransactionApi
+    path = url_for(StatusApi.__name__.lower(), tx_id=backlog_tx.id)
+    res = client.get(path=path)
+    assert res.json['status'] == Bigchain.TX_IN_BACKLOG
+    expected_tx_link = url_for(
+        TransactionApi.__name__.lower(), tx_id=backlog_tx.id)[7:]
```

Comment:
The `[7:]` is a bit obfuscated, maybe we could use a regex or strip delimited with `/`?